src/code/z_camera.c

```diff
@@ -749,34 +749,34 @@ s32 Camera_CopyPREGToModeValues(Camera* camera) {
     return true;
 }
 
-#define SHRINKWIN_MASK (0xF000)
-#define SHRINKWINVAL_MASK (0x7000)
-#define SHRINKWIN_CURVAL (0x8000)
+#define LETTERBOX_MASK (0xF000)
+#define LETTERBOX_SIZE (0x7000)
```

Comment:
not sure how exactly but I feel like this should keep "mask"
```suggestion
#define LETTERBOX_SIZE_MASK (0x7000)
```

or we leave it up to engineer's #1225 :)

Reply:
it will most likely change in that pr yeah, but I would like to remove all mentions of shrink_window (assuming we all actually agree on what this should be called eventually)